lib/okta/directory_service.rb

```diff
@@ -16,19 +14,23 @@ def scopes(category)
 
       headers = { apiKey: Settings.connected_apps_api.connected_apps.api_key }
 
-      response = RestClient::Request.execute(method: :get, url: scopes_url, headers:)
+      # modules/apps_api/app/controllers/apps_api/v0/directory_controller.rb
+      # expects to catch an error in controller
```

Comment:
I'm not sure I'd include this comment, especially because this service isn't coupled with the `directory_controller`. I would, however, include comments like this in the PR, so that if people are wondering, they couldn't trace back to the PR that made the change for context.